pkg/leetError/error.go

```diff
@@ -153,6 +155,7 @@ var (
 		InvalidServiceFeeError:       "An error occurred because the service fee is invalid",
 		RestrictedAccessError:        "User do not have authorization to access this endpoint",
 		FeesError:                    "There is an error with the application fees",
+		ResourceNotFoundError:        "An error occurred because the requested resource was not found",
```

Comment:
Update error code and message to be more specific to requested LGA

```suggestion
		LGANotFoundError:        "requested resource was not found",
```